doc/command-line.md

```diff
@@ -137,8 +140,12 @@ Options:
 ```
 --output -- save the distance matrix to this file (as a numpy binary matrix)
 --ksize -- do the comparisons at this k-mer size.
+--containment -- compute containment instead of similarity.
+        The value at `[i,j]` is the containment of `j` by `i`.
```

Comment:
would it be clearer if this was `C(i,j) = size(i intersection j) / size(i)`? The way this is phrased makes me think it is `C(i,j) = size(i intersection j) / size(j)`...

Reply:
fixed in 8b5c75e5, thx!